opensearchapi/api.indices.put_template.go

```diff
@@ -75,7 +75,7 @@ type IndicesPutTemplateRequest struct {
 }
 
 // Do executes the request and returns response or error.
-//
+// deprecated: uses a nonexistent legacy REST API (/_template) correct REST API is /_index_template use IndicesPutSettingsRequest instead
```

Comment:
If this API doesn't exist and doesn't work today, then there's no point in deprecating it, it should just be deleted.

Reply:
The API still works and it is valid to use it. The deprecation is correct.

Reply:
The comment is misleading then :)